js/apps/account-ui/package.json

```
@@ -1,12 +1,13 @@
 {
-  "name": "account-ui",
+  "name": "@keycloak/keycloak-account-ui",
+  "version": "0.0.5-alpha",
```

If this package is published we should use `999.0.0-SNAPSHOT` as a version, that will then be replaced by the actual release version when the release is ran.
```suggestion
  "version": "999.0.0-SNAPSHOT",
```